src/Analyzers/Core/CodeFixes/OrderModifiers/AbstractOrderModifiersCodeFixProvider.cs

```diff
@@ -58,8 +63,8 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         protected override async Task FixAllAsync(
             Document document, ImmutableArray<Diagnostic> diagnostics, SyntaxEditor editor, CancellationToken cancellationToken)
         {
-            var options = await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
-            var option = options.GetOption(_option);
+            var tree = await document.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
+            var option = document.Project.AnalyzerOptions.GetOption(_option, tree, cancellationToken);
```

Comment:
This is the only real semantic change in the PR - we now use a different approach for fetching option, which is available in both CodeStyle layer and Features layer.